Makefile

```
@@ -25,3 +25,13 @@ all install uninstall staged-install clean distclean maintainer-clean test check
 	   echo "You must use GNU make to build PostGIS." ; \
 	   false; \
 	 fi
+
+# This target must also work before configure, so dedicated CI can audit
+# contributor credits from a clean checkout with full Git history.
+check-contributor-credits:
```

checkmake (0.3.2)

[warning] Target body for "check-contributor-credits" exceeds allowed length of 5 lines (6).

(maxbodylength)